docs/source/contributing.rst

```diff
@@ -5,39 +5,28 @@ Welcome to the Kordac developer community! We have spent many months developing
 
 Kordac was created to be used by two much larger projects (the `CS Unplugged`_ and `CS Field Guide`_ websites) as the markdown-to-html converter. The tags we chose are designed to allow authors of these two projects to easily write material without technical elements getting in the way. It is therefore important to us that Kordac remains as simple and robust as possible, please keep this in mind if you decide to work on Kordac with us.
 
-.. _CS Unplugged: https://github.com/uccser/cs-unplugged/
-.. _CS Field Guide: https://github.com/uccser/cs-field-guide/
-
 The git repository for Kordac can be found `here`_, jump in and take a look around!
 
-.. _here: https://github.com/uccser/kordac
-
-<mention django>
```

Comment:
yea well I went through the code and didn't really think it was necessary to talk about it too much.
might add a short note now.